cmd/dfget/app/root.go

```diff
@@ -193,9 +193,9 @@ func initFlags() {
 	flagSet := rootCmd.Flags()
 
 	// url & output
-	flagSet.StringVarP(&cfg.URL, "url", "u", "", "URL of user requested downloading file(only HTTP/HTTPs supported)")
```

Comment:
Emm...
IMO, URL is a proper noun and it should always be capitalized. 
Ref: https://en.wikipedia.org/wiki/URL

Reply:
I agree with you